tutorials/maprapp/src/test/java/com/datatorrent/maprapp/ApplicationTest.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Put your copyright and license info here.
+ */
+package com.datatorrent.maprapp;
+
+import java.io.IOException;
+
+import javax.validation.ConstraintViolationException;
+
+import org.junit.Assert;
+
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Test;
+
+import com.datatorrent.api.LocalMode;
+import com.datatorrent.maprapp.Application;
+
+/**
+ * Test the DAG declaration in local mode.
+ */
+public class ApplicationTest {
+
+  @Test
+  public void testApplication() throws IOException, Exception {
+    try {
+      LocalMode lma = LocalMode.newInstance();
+      Configuration conf = new Configuration(false);
+      conf.addResource(this.getClass().getResourceAsStream("/META-INF/properties.xml"));
+      lma.prepareDAG(new Application(), conf);
+      LocalMode.Controller lc = lma.getController();
+      lc.run(10000); // runs for 10 seconds and quits
```

Comment:
Generally we recommend not doing this (see http://docs.datatorrent.com/beginner/#local-mode-testing).
Instead, use runAsync() and check for results as done in other examples.